forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -0,0 +1,190 @@
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the operation.
+     * On success, returns an object with `snap` (SNAP) and `cash` (EBT Cash) fields, whose values
+     * indicate the current balance of each respective tender.
+     *
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    override suspend fun checkBalance(
+        params: CheckBalanceParams
+    ): ForageApiResponse<String> {
+        return ForageApiResponse.Success("TODO")
+    }
+
+    // ======= Same as online-only Forage SDK below =======
+
+    /**
+     * Captures a Forage Payment associated with an EBT card using a customer's EBT card PIN.
+     *
+     * @param params The parameters required for payment capture, including
+     * reference to a ForagePINEditText and a Payment ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the
+     * payment capture. On success, returns a confirmation of the transaction.
+     * On failure, provides a detailed error response.
+     *
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    override suspend fun capturePayment(
+        params: CapturePaymentParams
+    ): ForageApiResponse<String> {
+        return forageSdk.capturePayment(params)
+    }
+
+    /**
+     * Collect's a customer's PIN for an EBT payment and defers
+     * the capture of the payment to the server.
+     *
+     * @param params The parameters required for deferring the capture of an EBT payment,
+     * including a reference to a ForagePINEditText and a Payment ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the
+     * PIN capture. On success, returns `Nothing`.
+     * On failure, provides a detailed error response.
+     *
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    override suspend fun deferPaymentCapture(params: DeferPaymentCaptureParams): ForageApiResponse<String> {
+        return forageSdk.deferPaymentCapture(params)
+    }
+
+    /**
+     * Use other `tokenizeEBTCard` methods with different signatures instead.
+     *
+     * @throws NotImplementedError
+     */
+    @Deprecated(
+        message = "This method is not applicable to the Forage Terminal SDK. Use the other tokenizeEBTCard methods.",
+        level = DeprecationLevel.ERROR
+    )
+    override suspend fun tokenizeEBTCard(params: TokenizeEBTCardParams): ForageApiResponse<String> {
+        throw NotImplementedError(
+            """
+            This method is not applicable to the Forage Terminal SDK.
+            Use the other tokenizeEBTCard methods.
+            """.trimIndent()
+        )
+    }
+}
```

Comment:
Ahh did you add this method because it's required for classes that implement `ForageSDKInterface`?

If so, I just want to make sure that we're on the same page: once you get to [FX-855](https://linear.app/joinforage/issue/FX-855/drop-reference-tokenizeebtcard-from-foragesdkinterface), we drop `tokenizeEBTCard` from `ForageSDKInterface`. At that point, I suspect we'd want to outright delete this method because the compiler won't enforce that `ForageTerminalSDK` declares a `tokenizeEBTCard` method anymore. Plus we'd avoid actually having to deal with depreciated a method that Toast never needed to use.

If it's easier, you even need to have `ForageTerminalSDK` implement the `ForageSDKInterface` right now. You can have `ForageTerminalSDK` extend the `ForageSDKInterface` after [FX-855](https://linear.app/joinforage/issue/FX-855/drop-reference-tokenizeebtcard-from-foragesdkinterface) is completed. Whatever you prefer!

Reply:
> Ahh did you add this method because it's required for classes that implement ForageSDKInterface?

If so, I just want to make sure that we're on the same page: once you get to FX-855, we drop tokenizeEBTCard from ForageSDKInterface


@devinmorgan 

Yep, that's the plan!